server/src/gameplay/game_event_handler.rs

```diff
@@ -271,6 +287,33 @@ pub fn update_game_state(game_state: &mut GameState, event: Event) -> Result<(),
             game_state.passed_players.insert(player);
             game_state.advance_turn()?;
         }
+        Event::ScorePublicObjective { player, objective } => {
+            // TODO: consider restricting to the correct phase, etc
```

Comment:
i'm leaning towards no, because i'm don't think we would gain anything from that

it we don't restrict it, we could re-use this event for "manually" scoring objectives outside the status phase.

also, i just realised, it should be legally possible to score a public objective outside of the status phase if you have the agenda that turns a secret objective into a public objective, right?